### DISCUSSION TITLE
These tools are gone

### DISCUSSION THREAD

**Discussion by lkxxxxx** [SkalskiP/make-sense#335] [Q&A]
The tool above the picture is missing, and there is no way to label the images.
![image](https://github.com/SkalskiP/make-sense/assets/101586594/1a1e1dfd-f3c5-4061-9d26-e49b61011734)
![image](https://github.com/SkalskiP/make-sense/assets/101586594/b0c95712-1a06-48db-a0e2-556a3cd2c07a)

**Comment by nicorl** [comment]
I use the tool regularly, and occasionally, I lose the cursor over the image (I believe this is what you mentioned in the picture). What I do to continue working is to click on "export annotations," and then I have the cursor back over the image, allowing me to continue labeling. There's no need to exit and reload the annotations, just click the export button.
